### PR TITLE
Add coloured image initialiser

### PR DESCRIPTION
**Pull request by elland**

### CODE REVIEW

**Comment by 3lvis** [comment]
😮 interesting. What's your use case for this?

**Comment by elland** [comment]
@3lvis creating background images for buttons, amongst other things. 👍

**Comment by elland** [comment]
Background image for table cells, too. 👍

File: Sources/UIImage+Sweetness.swift [hunk 1]
```diff
@@ -22,5 +22,17 @@ public extension UIImage {
 
         return centeredFrame
     }
+
+    public convenience init?(color: UIColor, size: CGSize = CGSize(width: 1, height: 1)) {
+        let rect = CGRect(origin: .zero, size: size)
```
Comment (3lvis) [inline Sources/UIImage+Sweetness.swift:27]: I think there was `CGRect.zero`. If that's the case could we just that here?

Comment (Elland) [inline Sources/UIImage+Sweetness.swift]: 😄 🔫

Comment (elland) [reply Sources/UIImage+Sweetness.swift:27]: @3lvis `[...] size: size)`

Comment (3lvis) [reply Sources/UIImage+Sweetness.swift:27]: I see :)

**Comment by 3lvis** [comment]
Would be nice if you could add some docs and ship it.

**Comment by 3lvis** [comment]
We could probably have unit tests for this as well.

**Comment by 3lvis** [comment]
Feel free to just make an issue and move on, though. Don't let things stop you from shipping 🎉

**Comment by elland** [comment]
@3lvis documented. 👍

**Comment by 3lvis** [comment]
@Elland Sweet, what about unit tests?

**Comment by elland** [comment]
@3lvis colour comparison is still all whack in Swift 3, no? How would I test this?

**Comment by 3lvis** [comment]
@Elland Add an image to the Xcode project and compare to the results of calling the method.

**Comment by 3lvis** [comment]
Or that wouldn't work 🤔

**Comment by elland** [comment]
@3lvis I don't think that works....

**Comment by 3lvis** [comment]
:Dolan:

**Comment by 3lvis** [comment]
Haha, okay then!

**Comment by 3lvis** [comment]
SHIIIIPPPP PPP P P P I ITITI ITITITI

**Pull request merged**